examples/tasks/run-mock-psutil.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+# get the directory the script exists in
+__dir="$(cd ../../scripts/docker/large && pwd)"
+__proj_dir="$(cd ../../ && pwd)"
+
+export PLUGIN_SRC="${__proj_dir}"
+
+# verifies dependencies and starts influxdb
+. "${__proj_dir}/examples/tasks/.setup.sh"
+
+# start the influxdb container
+(cd $__dir && docker-compose up)
+
```

Comment:
@candysmurf I'm not sure what do you mean. `make test-large` is enabled and starts large tests. Can you clarify what do you exactly mean?